plugins/modules/win_timezone.ps1

```diff
@@ -3,71 +3,75 @@
 # Copyright: (c) 2015, Phil Schwartz <schwartzmx@gmail.com>
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
-#Requires -Module Ansible.ModuleUtils.Legacy
+#AnsibleRequires -CSharpUtil Ansible.Basic
```

Comment:
This newer requires statement is used by Ansible to load in the `Ansible.Basic` C# module util.